.env.example

```diff
@@ -0,0 +1,4 @@
+REMINDER_DATA=
```

Comment:
what is this about?

Reply:
In the docker-compose.yml file you can see the REMINDER_DATA is used as an environment variable to describe where the volume of the redis database is located on the HOST system

I need this variable to make reminders persistent even if the docker container is destroyed.

Reply:
ah ok, so maybe rename it to `DOCKER_REDIS_VOLUME` to make its purpose clearer, and so people not using docker know not to worry about it.

Reply:
has been done